.github/CODEOWNERS

```diff
@@ -93,10 +93,10 @@
 # ServiceOwners:                                                   @trangevi @dargilco @jhakulin @glharper
 
 # PRLabel: %AI Projects
-/sdk/ai/Azure.AI.Projects                                          @jhakulin @nick863
+/sdk/ai/Azure.AI.Projects                                          @dargilco @jhakulin @nick863 @trangevi
```

Comment:
Looks like the `Microsoft` org membersship for `trangevi` is not marked as public.   We'll need to flip that over before merging.

Reply:
Can you elaborate on what that means? I don't see anywhere in my "organizations" where there's a way to mark membership as public or private. And I'm already listed in the codeowners file in other places, is this a new check?

Reply:
I see, alright I've flipped that to public,